pkg/controller/vpc.go

```diff
@@ -44,7 +44,7 @@ func (c *Controller) enqueueUpdateVpc(oldObj, newObj interface{}) {
 	oldVpc := oldObj.(*kubeovnv1.Vpc)
 	newVpc := newObj.(*kubeovnv1.Vpc)
 
-	if newVpc.DeletionTimestamp.IsZero() ||
+	if !newVpc.DeletionTimestamp.IsZero() ||
```

Comment:
maybe split the if !newVpc.DeletionTimestamp.IsZero() with the other is better.

if !newVpc.DeletionTimestamp.IsZero() {
// vpc is deleted
return nil

}

if ...(update while vpc is not deleted)